src/x-scale-axis.ts

```diff
@@ -47,19 +47,29 @@ export class AxisTicksSetter {
    * Handles setting the tickFormatter function
    */
   handleAxisTicks() {
-    const { scaleType } = this.cle;
-    if (scaleType !== "continuous" && scaleType !== "categorical") {
+    // NOTE: used to customize the tick values for certain legend types
+    if (
+      this.cle.scaleType !== "continuous" &&
+      this.cle.scaleType !== "categorical" &&
+      !this.cle.tickValues
+    ) {
       const [min, max] = this.xScale.domain() as [number, number];
-      this.cle.tickValues = this.cle.tickValues || [
+      this.cle.tickValues = [
         min,
         ...((this.cle.colorScale as ScaleQuantize<number>)?.thresholds?.() ||
           (this.cle.colorScale.domain() as number[])),
         max,
       ];
     }
-    if (this.cle.tickFormat?.length) {
+    // prefer `tickFormatter` property if it has been set
+    // TODO: how to override `tickFormat` after tickFormatter has previously been set and is already a function?
```

Comment:
code be a code smell that this should be handled separately in the main ColorLegendElement's class properties.